main.go

```diff
@@ -13,6 +13,15 @@ import (
 
 func main() {
 	log.SetFormatter(&log.TextFormatter{ForceColors: true})
+	log.SetOutput(colorable.NewColorableStderr())
+	msg := `
+aks-engine is deprecated for Azure public cloud customers.
```

Comment:
what do you think about linking to https://github.com/Azure/aks-engine#project-status for more info about timelines?

Reply:
Good idea. I wanted to keep this short and not include specifics (plans may change, but binaries live on...) but that would help.